filament/src/components/RenderableManager.cpp

```diff
@@ -67,9 +67,12 @@ struct RenderableManager::BuilderDetails {
     float2* mBoneIndicesAndWeight = nullptr;
     size_t  mBoneIndicesAndWeightsCount = 0;
 
+    std::map<int, utils::FixedCapacityVector<
```

Comment:
the key should probably be a `uint32_t` or `size_t` (it can't be < 0)

Reply:
Fixed, you are right